Pocket/pocket-ios
# feat(archive): improve handling of archive

## Summary

Improve the archive experience by allowing Archives to stay on the user's device.

In a follow-up we will update the store to clear out any old article (mArticle) data on archived content. IE Archived items will only have metadata stored locally.



## Implementation Details
* This merges the Saves and Archives view model because Archive is just a filter on saves.
    * This will allow us to implement features faster across Saves & Archives by not duplicating logic.
* Set the code to download 400 of the user's archives in a similar manner to how we download their saves. 

## Test Steps
* Open the app and ensure that Archives load from CoreData and do not download mArticle. This can be observed by clicking into an archived item and the reader view takes a few seconds to load from the server.

## PR Checklist:

## Screenshots
